docs/guides/hello.md

```diff
@@ -183,7 +183,7 @@ To install the gRPC runtime for your chosen language:
   </ul>
 
 <div id="java_install">
-<p>This example is also part of Java gRPC itself's build, so running it is a little different to creating and running a regular project. To install and build it follow the instructions in the <a href="https://github.com/grpc/grpc-java/tree/master/examples">Quickstart</a>.</p>
+<p>This example is also part of Java gRPC's build, so running it is a little different to creating and running a regular project. To install and build it follow the instructions in the <a href="https://github.com/grpc/grpc-java/tree/{{site.data.config.grpc_release_branch}}/examples">Quickstart</a>.</p>
```

Comment:
This paragraph needs to be fixed a bit more. The example is no longer part of Java's build. I'll need to have a followup PR.